### PR TITLE
TechPage created.

### PR DESCRIPTION
**Pull request by raeioul**
Please review.

techpage_upload

### CODE REVIEW

File: src/pages/containers/TechPage.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import { TechIconList } from '@nearshorecode/web-ui';
+import './TechPage.css';
+
+const padding = {padding:'30px'}
```
Comment (diegolanda) [inline src/pages/containers/TechPage.js]: const ICON_STYLE = { padding: '30px' }

File: src/pages/containers/TechPage.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          <p>For the past 4 years NearShore Code has been providing software development services to businesses of differents</p>
+          <p>areas and sizes.Thus, we have a range of specialist experienced in JS technologies, .Net Core and Mobile.</p>
+          <p>Over time, we demonstrated capability to develop applications with the best quality.</p>
+      </div>
+      ),
+      (
+      <div className="tech-page-text">
+        <h1>JS TECHNOLOGY</h1>  
+        <div  className="tech-page-container">
+          <TechIconList icons={jsTechnologies} flow="horizontal"/>
```
Comment (diegolanda) [inline src/pages/containers/TechPage.js]: jsTechnologies.map(x => {...x, ICON_STYLE})

File: src/pages/containers/TechPage.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+const jsTechnologies = [
+  {name:'angular', style: padding},
+  {name:'react', style: padding},
+  {name:'js', style: padding},
+  {name:'node', style: padding},
+  {name:'ionic', style: padding},
+  {name:'redux', style: padding},
+  {name:'meteor', style: padding},
+];
+const dbTechnologies = [
```
Comment (diegolanda) [inline src/pages/containers/TechPage.js]: all caps

File: src/pages/containers/TechPage.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  {name:'pgdb', style: padding},
+  {name:'mysql', style: padding},
+  {name:'mongodb', style: padding },
+  {name:'ravendb', style: padding },
+  {name:'sqlserver', style: padding },
+];
+const mobileTechnologies = [
+    {name:'xamarin', style: padding},
```
Comment (diegolanda) [inline src/pages/containers/TechPage.js]: remove `style: padding`

File: src/pages/containers/TechPage.js (hunk 1 above)
Comment (raeioul) [reply src/pages/containers/TechPage.js]: Fixed

File: src/pages/containers/TechPage.js (hunk 2 above)
Comment (raeioul) [reply src/pages/containers/TechPage.js]: Done.

File: src/pages/containers/TechPage.js (hunk 3 above)
Comment (raeioul) [reply src/pages/containers/TechPage.js]: Fixed.

**Pull request merged**